testplan/common/utils/sockets/fix/server.py

```diff
@@ -20,7 +20,7 @@ class ConnectionDetails(object):
     """
 
     def __init__(
-        self, connection, name=None, queue=None, in_seqno=1, out_seqno=1
+        self, connection, name=None, msg_queue=None, in_seqno=1, out_seqno=1
```

Comment:
no need to change the public api for this fix, as some might use it with keyword args. It can just remains queue in the constructor, it is not a problem if that hides the package.